roles/pulsar-post-tasks/tasks/main.yml

```diff
@@ -0,0 +1,27 @@
+---
+- name: Install slurm-drmaa
+  package:
+    name: slurm-drmaa1
+    state: present
+- name: own the /mnt/pulsar dirs as ubuntu
+  file:
+      path: /mnt/pulsar
```

Comment:
Change these to the variable for pulsar root...

Reply:
good thinking